Dockerfile

```diff
@@ -1,11 +1,17 @@
-FROM node:6.10.3-alpine
+FROM node:10.14.2-alpine
+
+RUN apk --no-cache update && \
+    apk --no-cache upgrade
 
 WORKDIR /app
 
-COPY . .
+COPY package.json package.json
 
-RUN yarn install
+RUN yarn install && \
+    yarn cache clean
```

Comment:
travis use npm and not yarn. Usually it is not recommended to use both program on the same project.